test/spec/f16.wast

```diff
@@ -25,3 +31,53 @@
 (assert_return (invoke "f16x8.extract_lane_last" (v128.const i16x8 0 0 0 0 0 0 0 0xc500)) (f32.const -5))
 (assert_return (invoke "f16x8.replace_lane_first" (v128.const i64x2 0 0) (f32.const 100.5)) (v128.const i16x8 0x5648 0 0 0 0 0 0 0))
 (assert_return (invoke "f16x8.replace_lane_last" (v128.const i64x2 0 0) (f32.const 100.5)) (v128.const i16x8 0 0 0 0 0 0 0 0x5648))
+
+;; comparisons
+(assert_return (invoke "f16x8.eq"
+    ;;                0 -1     1      nan    inf    nan    inf    nan
+    ;;                0 0      -1     nan    inf    0      -inf   inf
+    (v128.const i16x8 0 0xbc00 0x3c00 0x7e00 0x7c00 0x7e00 0x7c00 0x7e00)
```

Comment:
Does the proposal include a new way to write `v128.const f16x8 ...` in the text format? If not, it should, although obviously that's not the highest priority feature right now.

Reply:
It doesn't currently. I filed an issues about this though https://github.com/WebAssembly/half-precision/issues/6